Kanbas/quizzes/dao.js

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose";
+import model from "./model.js";
+
+export const getAllQuizzesByCourseId = (courseId) =>
+    model.find({ course: courseId });
+
+export const createQuiz = (courseId, quiz) => {
+    delete quiz._id; // delete id if being passed by UI 
+    quiz.course = mongoose.Types.ObjectId.createFromHexString(courseId);
+    return model.create(quiz);
+}
+
+export const deleteQuiz = (quizId) =>
+    model.deleteOne({ _id: mongoose.Types.ObjectId.createFromHexString(quizId) });
```

Comment:
this can be given as "quizId", so that the translation to `ObjectId` doesn't have to be done here.

Reply:
Enforcing this in BE is not required either.